contentgrid-spring-querydsl/src/main/java/com/contentgrid/spring/querydsl/predicate/Text.java

```diff
@@ -160,27 +161,60 @@ public Optional<Expression<? extends Comparable<?>>> sortExpression(Path<?> path
      * <p>
      * This predicate only supports {@link String}s, and can not be used with other types.
      */
+    public static class StartsWithNormalized extends AbstractStringPredicateFactory {
+
+        protected StartsWithNormalized() {
+            super((expr, value) -> postgresNormalize(expr).startsWith(Normalizer.normalize(value, Form.NFKC)));
+        }
+    }
+
+    /**
+     * Filters items down to only items starting with the supplied value in a case-insensitive, NFKC normalized way.
+     * <p>
+     * This predicate only supports {@link String}s, and can not be used with other types.
+     */
     public static class StartsWithIgnoreCaseNormalized extends AbstractStringPredicateFactory {
 
         protected StartsWithIgnoreCaseNormalized() {
-            super((expr, value) -> normalize(expr).startsWithIgnoreCase(Normalizer.normalize(value, Form.NFKC)));
+            super((expr, value) -> postgresNormalize(expr).startsWithIgnoreCase(Normalizer.normalize(value, Form.NFKC)));
         }
     }
 
     /**
-     * Filters items down to only items starting with the supplied value in a case-insensitive, NFKC normalized way.
+     * Filters items down to only items starting with the supplied value
+     * in a case-insensitive, accent-insensitive, NFKC normalized way.
      * <p>
+     * Requires Postgres extension {@code unaccent} and a function named
+     * {@code contentgrid_prefix_search_normalize} defined in a schema named {@code extensions}:
+     * <pre><code>
+     * CREATE SCHEMA extensions;
+     * CREATE EXTENSION unaccent SCHEMA extensions;
+     * CREATE OR REPLACE FUNCTION extensions.contentgrid_prefix_search_normalize(arg text)
+     *   RETURNS text
+     *   LANGUAGE sql IMMUTABLE PARALLEL SAFE STRICT
+     * RETURN extensions.unaccent('extensions.unaccent', lower(normalize(arg, NFKC)));
+     * </code></pre>
      * This predicate only supports {@link String}s, and can not be used with other types.
      */
-    public static class StartsWithNormalized extends AbstractStringPredicateFactory {
+    public static class StartsWithIgnoreCaseAccentNormalized extends AbstractStringPredicateFactory {
```

Comment:
```suggestion
    public static class ContentGridPrefixSearch extends AbstractStringPredicateFactory {
```

I would call all things related to this functionality `ContentGridPrefixSearch`, because the naming of this predicate factory is getting a bit ridiculous and unreadable